docs/Developer-guide/Building-GlusterFS.md

```diff
@@ -249,8 +249,7 @@ steps to build RPMs,
 
 This will create rpms from the source in 'extras/LinuxRPM'. *(Note: You
 will need to install the rpmbuild requirements including rpmbuild and
-mock)
-
+mock)<br>
```

Comment:
I don't see the need for removing * and adding <br>, if you still wish to do so please remove * before `(Note:` as well

Reply:
https://github.com/gluster/glusterdocs/blob/35d17d7355a6223aec308e0d928f5d8f488d3288/docs/Developer-guide/Building-GlusterFS.md 

The above comment is valid. Adding <br> at end can happen with a '*' too. But without that, the highlighting goes for a toss!